### ISSUE TITLE
Results never changed

### ISSUE THREAD

**Issue by BlankAix** [hizhangp/yolo_tensorflow#41]
Whatever photos I put , the net_ouput will be the same.And the final image showed last have the same frame.I am upset about this phenomenon.
Hope someone could tell me any advice.
Thank you.

[ ] def detect_from_cvmat(self, inputs):
[ ]     net_output = self.sess.run(self.net.logits,
[ ]                                feed_dict={self.net.images: inputs})

**Comment by hizhangp** [comment]
Which image did it show?

**Comment by BlankAix** [comment]
@hizhangp  I figured out that I hadn't use the original weight  you trained. Finally, I changed two of the class names in order to train my objectives including the ball and the cube. But I still confused because I don't know how to train only two classes with your weight. Anyway, thanks for your contribution.

**Comment by XiangqianMa** [comment]
I think you can not train your data with the author's weight, because this weight is suit for this special net which classes number is 20.

**Comment by lei9wang** [comment]
I have a issue, why the author's weight only have one file "YOLO_small.ckpt"
but there are three files "model.cpkt.index，model.cpkt.data and model.cpkt.meta" when we train our own generation?
is anybody know why?

**Comment by XiangqianMa** [comment]
It's because of the version of tensorflow, 1.* rather than 0.* . @lei9wang

**Comment by lei9wang** [comment]
@XiangqianMa 
yes, I got it 
注：Tensorflow 0.11版本以前，Checkpoint file只有一个后缀名为.ckpt的文件。
Thank you for your information.
